### PR TITLE
Validation + --create-organization

### PR DESCRIPTION
**Pull request by jdartigalongue**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/923#" title="Author self-approved">jdartigalongue</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [jdartigalongue]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_create.go [hunk 1]
```diff
@@ -663,9 +670,33 @@ func updatePolarisSpecWithFlags(cmd *cobra.Command, namespace string) (*polaris.
 	}
 	polarisSpec.Namespace = namespace
 
+	if err := validatePolaris(polarisSpec); err != nil {
+		return nil, err
+	}
 	return &polarisSpec, nil
 }
 
+func validatePolaris(polarisConf polaris.Polaris) error {
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_create.go]: Can you pls add all the validation in Args method?

Args: func(cmd *cobra.Command, args []string) error {
		// Check the Number of Arguments
		if len(args) != 1 {
			cmd.Help()
			return fmt.Errorf("this command takes 1 argument")
		}
		checkPasswords(cmd.Flags())
		return nil
	},

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/923#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/923#issuecomment-537228867" title="LGTM">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [jdartigalongue,msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**